packages/falcon-shop-data/src/Account/RequestPasswordResetMutation.tsx

```diff
@@ -0,0 +1,19 @@
+import gql from 'graphql-tag';
+import { Mutation } from 'react-apollo';
+import { EmailInput } from '@deity/falcon-shop-extension';
+
+export const REQUEST_CUSTOMER_PASSWORD_RESET_TOKEN_MUTATION = gql`
+  mutation RequestCustomerPasswordResetToken($input: EmailInput!) {
```

Comment:
I am not convinced that we should use `EmailInput` here, I think we should create `RequestCustomerPasswordResetTokenInput` type instead:
```
export type RequestCustomerPasswordResetTokenInput = {
   email: string;
}
```

```suggestion
  mutation RequestCustomerPasswordResetToken($input: RequestCustomerPasswordResetTokenInput!) {
```

Reply:
yep, good point - done.